pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -235,11 +163,12 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 			// TODO: I don't think there's a need for this if we've stored the
 			// entirety of the data previously & can iterate through the slice
 			// directly. Might consider refactoring this later.
-			repoInfo := &Repo{
-				Name:          name,
-				Owner:         ghs.cfg.GitHubOrg,
-				DefaultBranch: defaultBranch,
-			}
+
```

Comment:
if this commented out snippet ( & comment above it ) aren't being used we can go ahead and remove entirely